utils/argumentParser.js

```diff
@@ -377,9 +377,9 @@ function testEnv() {
                ______aBC______123______: 121,
                _______aBC_______123_______: 122
               };
-  console.log('Input = '+JSON.stringify(input, null, 2));
+  console.log('ZWED0139I - Input = '+JSON.stringify(input, null, 2));
   let output = EnvironmentVarsToObject(undefined, input);
-  console.log('Output = '+JSON.stringify(output, null, 2));
+  console.log('ZWED0140I - Output = '+JSON.stringify(output, null, 2));
```

Comment:
Doesn't show up in the JSON, if you are trying to keep it all in one place.

Reply:
corrected.